internal/switcher/interfaces.go

```diff
@@ -53,5 +53,5 @@ func (a InterfacesApplier) getTpl() *template.Template {
 	if a.tplFile != "" {
 		return template.Must(template.New(path.Base(a.tplFile)).ParseFiles(a.tplFile))
 	}
-	return template.Must(template.New("interfaces.tpl").Parse(interfacesTPL))
+	return template.Must(template.New("interfaces.tpl").Parse(mustReadTpl("interfaces.tpl")))
```

Comment:
I think I'd still prefer defining the template as a variable, which can be initialized using your new `mustReadTpl` function. The template does never change anyway and then the program fails on initialization and not at some point during runtime.

Reply:
good point